docs/build/tutorials/nodes-and-staking/set-up-an-avalanche-node-with-google-cloud-platform.md

```diff
@@ -0,0 +1,139 @@
+---
+description: 'Tutorial generated by community Member: willwhite123'
```

Comment:
Description should be used for a short description of the contents of the tutorial (this is used in SEO, for example).

The existing short note about you in the main body of the tutorial is enough, you may add a link to your twitter, github or something if you want.